ansible/destroy.yml

```diff
@@ -20,8 +20,13 @@
 
 - import_playbook: >-
     {{ lookup('first_found', {
-         'files': [ 'destroy_env.yml', cloud_provider + '_destroy_env.yml' ],
-         'paths': [ 'configs/' + env_type, 'cloud_providers/' ]
+         'files': [ 'destroy_env.yml',
+                    'destroy_env_' + cloud_provider + '.yml',
```

Comment:
I'm not opposed to this change...but rather than making the change here, can't we just leave the generic `destroy_env.yml` and import the right set of tasks on conditional? That follows the pattern we use in many other places.

If ec2, import destroy_env_ec2.yml
If osp, import destroy_env_osp.yml
etc.

Reply:
Hm. That's not a bad idea. I'll change and test.